scripts/install

```diff
@@ -96,43 +96,26 @@ phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
 #=================================================
 # INSTALL COMPOSER AND DEPENDENCIES
 #=================================================
-ynh_script_progression --message="Installing composer and dependencies..."
+ynh_script_progression --message="Installing Composer and dependencies..."
 
 ynh_install_composer --phpversion="$phpversion" --workdir="$final_path"
 
+# Install missing icons
+git clone https://github.com/matomo-org/matomo-icons.git "$final_path/plugins/Morpheus/icons" --quiet
```

Comment:
why not using something like ynh_setup_source --dest_dir="$final_path/plugins/Morpheus/icons" --source_id="icons" adn having an icons.src file ?